src/index.js

```diff
@@ -0,0 +1,60 @@
+import './style.css';
+import { getTodos } from './modules/store.js';
+import TodosContainer from './modules/TodosContainer.js';
+import { updateStatus, clearTodoList } from './modules/updateStatus.js';
+import UI from './modules/UI.js';
+
+const todoLists = getTodos() || [];
+const container = new TodosContainer();
+
+const form = document.querySelector('.task-adder');
+const todoElements = document.querySelector('.tasks');
+const { task } = form.elements;
+const clearButton = document.querySelector('.button-wrapper button');
+function populateTodos(data) {
+  data.forEach((element) => {
+    UI.add(element.id, element.description, element.completed);
+  });
+}
+form.addEventListener('submit', (event) => {
```

Comment:
In the event listener, the logic to toggle the completed status of a todo item could be extracted into a separate function instead of being repeated in two different event listeners then the function can be reused in both event listeners. In order to avoid repeating yourself

Reply:
Thanks for the review @Lornakaboro , I will do that🙏